asn1/asn1.go

```diff
@@ -936,15 +936,15 @@ func parseField(v reflect.Value, bytes []byte, initOffset int, params fieldParam
 		err = err1
 		return
 	case timeType:
-		var time time.Time
+		var tm time.Time
 		var err1 error
```

Comment:
I wonder why `err1` exists, when they do `err = err1` at the bottom of this block anyway?

Reply:
Reverted. The err1 thing isn't immediately obvious, SEP field on though.

Reply:
Why did you revert this change?